toast-node-wrapper/toast-render.mjs

```diff
@@ -29,7 +29,9 @@ async function main() {
     const wrapper = await import(pageWrapperPath);
     pageWrapper = wrapper.default;
   } catch (e) {
-    console.error("no user pagewrapper supplied", e);
+    if (e.code !== "ERR_MODULE_NOT_FOUND") {
```

Comment:
This ignores `MODULE_NOT_FOUND` for imports within the page-wrapper as well, correct? I believe we need to do some checking of the message as well to make sure it's a `MODULE_NOT_FOUND` for the page-wrapper and not some arbitrary import inside of the page wrapper.

Reply:
Good call! Fixed! Now it checks for page-wrapper file's existence before importing it.:
Non existent import in `page-wrapper` will now throw:
![image](https://user-images.githubusercontent.com/22195362/95515071-ab83e380-09da-11eb-892e-e87f5f5ccf8b.png)